crates/solidity/inputs/language/src/definition.rs

```diff
@@ -2817,16 +2817,19 @@ codegen_language_macros::compile!(Language(
                                 EnumVariant(name = String, reference = StringKeyword),
                                 EnumVariant(name = Address, reference = AddressType),
                                 EnumVariant(name = Payable, reference = PayableKeyword),
-                                EnumVariant(name = ByteArray, reference = BytesKeyword),
-                                EnumVariant(name = SignedInteger, reference = IntKeyword),
-                                EnumVariant(name = UnsignedInteger, reference = UintKeyword),
+                                EnumVariant(name = ByteArray, reference = FixedBytesType),
+                                EnumVariant(name = SignedInteger, reference = SignedIntegerType),
+                                EnumVariant(
```

Comment:
I'm fine with the name change, if it is more idiomatic.
But if we now consider them types, not keywords (from a spec POV), should we also move them to the end of `Elementary Types` topic, and out of the `Keywords` topic?